### PR TITLE
Feature: Doctors search and filter screens

### LINKED ISSUE #19: Feature:  Doctors screens
Doctors screens (screen 28 and 30)

Description

A patient will be able to find doctors online and twhat they are specialized and the hospitals they work from
People will also make doctors they want their favorites and they can also remove them from their favorites list

### Acceptance Criteria

 A person using this app can make a doctor his/her favorite
 Aperson using the app can remove a doctor from his/her  favorite list  
 A person can give reviews to the doctor 
A person can filter  doctors using their names, roles and reviews

### PR DESCRIPTION
**Pull request by Latiah**
What does this PR do?

This PR includes designs of favorite doctors, top doctors and list of all doctors where you can filter and find any doctor registered in the system of the app

Description of Task to be completed?

Add photos of the doctors and their information including reveiws and the hospitals their work on and their roles
create needed components for the designs like the cards ,buttons

How should this be manually tested?

You will click on the heart icon on the Home screen to go to favorite screen and on the favorite doctor screen then click on one of the cards where you will go to a screen of removing a doctor from the favoritates .On the home screen too you will click on the see all which is next to top doctors and you will be redirected to the top doctor screen ,there you will click on the search icon where you will be directed to the search screen which will load and after sometime you will be redirected to the not found screen and on the not found you can click on deafult you will be directed to a page where doctors were found and their list and when you click on default you will be directed to the filter screen .

Any background context you want to provide?
N/A

What are the relevant pivotal tracker/Trello stories?

Screenshots of the designs in this PR

Screenshot 2024-05-10 at 14 00 08

Screenshot 2024-05-10 at 14 00 08

Screenshot 2024-05-10 at 14 00 08

Screenshot 2024-05-10 at 14 00 08
Screenshot 2024-05-10 at 14 00 08

Screenshot 2024-05-10 at 14 00 08

Screenshot 2024-05-10 at 14 00 08

### CODE REVIEW

**Review by justinmihigo — APPROVED** [review]
Your screens looks good but the commit message does not
 **squashed commits**  you should use imperative sentences and type of the task you did for example this should be like if it is a chore, fix, feature and use the commit subject like **squash commits** instead of squashed.
**insert screenshot in the descripiton of your PR**

**Comment by Latiah** [comment]
Thanks @justinmihigo  let me change that

**Review by Adadie — APPROVED** [review]
1. Filter should display when someone clicks the filter icon in the search bar
2. Ensure that all your screens are aligned in the center on all devices

File: app/(tabs)/index.tsx [hunk 1]
```diff
@@ -157,7 +162,10 @@ const Home = () => {
       <View className="flex-row items-center w-full justify-between px-6 mb-3">
         <Text className="text-[20px] font-['UrbanistBold']">Top Doctors</Text>
         <TouchableOpacity activeOpacity={0.8}>
-          <Text className="text-[16px] font-['UrbanistBold'] text-lightblue">
```
Comment (chrysloi) [inline app/(tabs)/index.tsx:160]: Consider keeping this line the styling on this kept won't work as expected.

Comment (Latiah) [reply app/(tabs)/index.tsx:160]: I will in the fix pr

**Pull request merged**